src/app/[narmesteLederId]/aktiv-plan/page.tsx

```diff
@@ -0,0 +1,33 @@
+import { Suspense } from "react";
+import {
+  NYLIG_OPPRETTET_SEARCH_PARAM,
+  getAGOversiktHref,
+} from "@/common/route-hrefs";
+import AktivPlanForAG from "@/components/FerdigstiltPlanSider/AktivPlanSide/AktivPlanForAG";
+import FerdigstiltPlanSkeleton from "@/components/FerdigstiltPlanSider/FerdigstiltPlanSkeleton";
+import Breadcrumbs from "@/ui/Breadcrumbs";
+
+export default async function AktivPlanPageForAG({
```

Comment:
Merker jeg er litt lite glad i prefixer og postfixer for AG og SM. Hadde det gått an å droppe det (sånn jevnt over), og heller være ganske tydelig på inndeling i mappestruktur/komposisjon?

Ikke noe stor krise, i alle fall ikke for denne filen her, men om vi har mange SM og AG ting i navngiving så er jeg litt redd for at vi blander for mye / ikke skiller SM og AG godt nok